pkg/cmd/cli/describe/describer.go

```diff
@@ -361,11 +345,8 @@ func (d *BuildConfigDescriber) Describe(namespace, name string) (string, error)
 			return nil
 		}
 		fmt.Fprintf(out, "Builds:\n  Name\tStatus\tDuration\tCreation Time\n")
-		sortedBuilds := sortableBuilds(builds.Items)
```

Comment:
We don't need to declare a new variable here. It is a slice, the contents are not copied but just referenced. Changing (sorting) `sortedBuilds` also affects `builds.Items`. http://play.golang.org/p/Tk0FbAvk_W